Index: opencv/cmake/OpenCVDetectOpenCL.cmake
===================================================================
--- opencv.orig/cmake/OpenCVDetectOpenCL.cmake	2024-12-11 01:59:52.939054649 +0000
+++ opencv/cmake/OpenCVDetectOpenCL.cmake	2024-12-11 01:59:52.935054761 +0000
@@ -5,7 +5,6 @@
 else()
   set(OPENCL_LIBRARY "" CACHE STRING "OpenCL library")
   set(OPENCL_INCLUDE_DIR "${OpenCV_SOURCE_DIR}/3rdparty/include/opencl/1.2" CACHE PATH "OpenCL include directory")
-  ocv_install_3rdparty_licenses(opencl-headers "${OpenCV_SOURCE_DIR}/3rdparty/include/opencl/LICENSE.txt")
 endif()
 mark_as_advanced(OPENCL_INCLUDE_DIR OPENCL_LIBRARY)
 
